app/jobs/import_twilio_data_job.rb

```diff
@@ -0,0 +1,25 @@
+class ImportTwilioDataJob < ActiveJob::Base
+  class InvalidDateParameter < StandardError
+    def initialize(msg, backtrace = nil)
+      set_backtrace(backtrace) if backtrace
+      super(msg)
+    end
+  end
+
+  queue_as :default
+
+  rescue_from(InvalidDateParameter) do |exception|
+    Bugsnag.notify(exception)
+  end
+
+  def perform(date_string)
+    date = parse_date(date_string)
+    DailyCalls::Twilio::Importer.new.import(start_date: date, end_date: date)
+  end
+
+  def parse_date(date_string)
+    Date.parse(date_string)
```

Comment:
We can, the issue is that we open ourselves up to swallowing Argument errors that occur during the actual processing so they don't get automatically rescheduled.

Can see the logic in simplifying, I would still try and include the date string as that gives us a starting point to what went wrong.